examples/jvm/jvm-cfl-example.yaml

```diff
@@ -1,15 +1,18 @@
 apiVersion: chaos-mesh.org/v1alpha1
-kind: JVMChaos
+kind: Schedule
 metadata:
   name: jvm-cfl-example
   namespace: app
 spec:
-  action: cfl
-  target: jvm
-  mode: one
-  selector:
-    labelSelectors:
-      app: springboot-jvmchaos-demo
-  duration: "30s"
-  scheduler:
-    cron: "@every 1m"
+  schedule: '@every 1m'
+  type: JVMChaos
+  historyLimit: 5
+  concurrencyPolicy: Forbid
+  jvm_chaos:
```

Comment:
change  "jvm_chaos"  to "jVMChaos" ?    There are similar situations in the following yaml files, so I will not list them one by one.